packages/web/components/cards/rewards-card.tsx

```diff
@@ -8,9 +8,13 @@ export const RewardsCard: React.FC<{
   title: string;
   titleIconUrl: string;
   tooltipContent: string;
-}> = ({ title, titleIconUrl, tooltipContent }) => {
+  onClick: () => void;
+}> = ({ title, titleIconUrl, tooltipContent, onClick }) => {
   return (
-    <div className="flex w-full flex-grow flex-col rounded-xl border-2 border-osmoverse-600">
+    <div
+      className="flex w-full flex-grow cursor-pointer flex-col rounded-xl border-2 border-osmoverse-600"
+      onClick={onClick}
+    >
```

Comment:
It appears this is a redundantly nested div? Let me know what that is for, perhaps for the border?